### PR TITLE
English review for Installing Tizen Studio page

### PR DESCRIPTION
**Pull request by divyk**
### Change Description ###

Made minor changes in the content. Please review.

### CODE REVIEW

File: docs/application/tizen-studio/setup/install-sdk.md [hunk 1]
```diff
@@ -1,40 +1,36 @@
...
 ## Using the GUI Installer
 
 To install the Tizen Studio:
 
 1. Launch the Tizen Studio installer.
 
-2. Accept the software license.
-
-   The license contains important legal notices for using the Tizen Studio. Read it fully, and click **Accept** only if you agree with the license statement.
+2. Accept the software license by reading the entire agreement, and click **Accept** only if you agree with the license statement:
```
Comment (journey2w) [inline docs/application/tizen-studio/setup/install-sdk.md]: I think the original sentences are better than your updates.

File: docs/application/tizen-studio/setup/install-sdk.md [hunk 2]
```diff
@@ -1,40 +1,36 @@
...
 
    ![Tizen Studio License Agreement](./media/install_sdk_license.png)
 
-3. Set the SDK and data location.
-
-   The Tizen Studio and its data directory are installed in the location specified in the Installation settings step. To change the directory, click the **...** button and specify a new directory. If the new directory is valid (no errors are shown), click **Install** at the bottom of the installer.
+3. Click ![Button](./media/advanced_conf_browse.png) and specify a new directory to set the SDK and data location. If the new directory is valid and shows no errors, click **Install**:
```
Comment (journey2w) [inline docs/application/tizen-studio/setup/install-sdk.md]: I think text is better than image to maintain docs. Please use text: **...** button.

File: docs/application/tizen-studio/setup/install-sdk.md [hunk 3]
```diff
@@ -1,40 +1,36 @@
...
 
-5. Install additional tools.
+5. Click **Finish** and close the installer:
 
-   After the installation is finished, you can close the installer by clicking **Finish**.
 
    ![Installation complete](./media/migration_finish_instal.png)
 
-   If you want to install additional platforms and tools, launch the Package Manager by checking the **Launch the Package Manager** check box before clicking **Finish**.
+   To install additional platforms and tools, launch the Package Manager by selecting the **Launch the Package Manager** check box and click **Finish**.
```
Comment (journey2w) [inline docs/application/tizen-studio/setup/install-sdk.md]: I think the original sentence is better than your change.

**Comment by huytaduc** [comment]
checkout the webpage https://docs.stg.tizen.org/staging/488

File: docs/application/tizen-studio/setup/install-sdk.md [hunk 4]
```diff
@@ -12,29 +12,27 @@ To install the Tizen Studio:
 
 2. Accept the software license.
 
-   The license contains important legal notices for using the Tizen Studio. Read it fully, and click **Accept** only if you agree with the license statement.
+   The license contains important legal notices for using Tizen Studio. Read it fully, and click **Accept** only if you agree with the  license statement:
```
Comment (journey2w) [inline docs/application/tizen-studio/setup/install-sdk.md]: There are two spaces in front of "license". The HTML render remove a space when it generates a HTML file. Nevertheless, you had better remove a space.

**Comment by huytaduc** [comment]
checkout the webpage https://docs.stg.tizen.org/staging/488

**Pull request merged**